indico/types/model.py

```diff
@@ -29,3 +29,16 @@ class Model(BaseType):
     id: int
     status: str
     training_progress: TrainingProgress
+
+
+class ModelOptions(BaseType):
+    """
+    Model Options in the Indico Platform.
+
+    Attributes:
+        id (int): The model options id
+        model_training_options (str): JSONString representation of the model training options specified
+    """
+
+    id: int
+    model_training_options: str
```

Comment:
should this be dict or None? and should we deserialize json here?

Reply:
making it dict or None is causing issues in BaseType: `TypeError: issubclass() arg 1 must be a class`

Opting to make it just dict for now